i18n-embed/src/fluent/mod.rs

```diff
@@ -113,7 +118,7 @@ impl FluentLanguageLoader {
     pub fn get_args_concrete<'source>(
         &self,
         message_id: &str,
-        args: HashMap<&'source str, FluentValue<'source>>,
+        args: HashMap<Cow<'source, str>, FluentValue<'source>>,
```

Comment:
This is a breaking change (see fl_macro failing unit test), was there are particular motivation for using `Cow` over `&str`?